### PR TITLE
docs: Put yarn in front of npm

### PR DESCRIPTION
**Pull request by ludanxer**
**Summary**

Ref: #1936 #1286 

If you think this is a good idea, I will finish the corresponding Chinese document.

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: packages/docs/docs/guide/deploy.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ You can also run the above script in your CI setup to enable automatic deploymen
 
 2. Create a file named `.travis.yml` in the root of your project.
 
-3. Run `npm install` locally and commit `package-lock.json` as it’s required for `npm ci`.
+3. (Only for npm users) Run `npm install` locally and commit `package-lock.json` as it’s required for `npm ci`.
```
Comment (kefranabg) [inline packages/docs/docs/guide/deploy.md]:
```suggestion
3. Run `yarn` or `npm install` locally and commit the generated `lock file`.
```

**Comment by ludanxer** [comment]
Chinese translation is done.

**Comment by meteorlxy** [comment]
I'm not sure if it's necessary to change the deploying guide.

It seems that a warning tip in the plugin guide is enough?

**Comment by ludanxer** [comment]
Actually, I'm not very sure either.

But I think it would be a little more user-friendly for beginners and easier to copy and paste?

**Pull request merged**